tests-arquillian/src/test/java/org/jboss/weld/tests/interceptors/producer/InterceptionFactoryTest.java

```diff
@@ -65,6 +66,13 @@ public void testFooClassLevelBinding(@Produced("classLevel") Foo foo) {
     public void testFooEjbInterceptors(@Produced("ejbInterceptors") Foo foo) {
         assertEquals("Hello pong", foo.ping());
     }
+    
+    @Test
+    public void testFooNoInterceptors(@Produced("empty") Foo foo) {
+        // Note that the produced Foo is @Dependent
+        assertEquals(Foo.class, foo.getClass());;
+        assertEquals("pong", foo.ping());
```

Comment:
@mkouba I don't see how this verifies there is no proxy created?
The fact that you get `pong` string back is derived from the mere absence of **any** interceptor binding. It does not mean you don't have the proxy created, right?